runtime/legion/legion_ops.cc

```diff
@@ -12803,6 +13014,7 @@ namespace Legion {
         assert(value != NULL);
 #endif
         InstanceSet mapped_instances;
+        // TODO: Implement physical tracing for fill operation
```

Comment:
TODO for the github list

Reply:
I don't think this comment is even true anymore.